PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -117,8 +125,8 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   /// \param collision collision to test against the selection criteria
   /// \param centrality collision centrality variable to be set in this function
   /// \return bitmask with the event selection criteria not satisfied by the collision
-  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll>
-  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality)
+  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll, typename BCs>
+  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, const BCs& /*bcs*/, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb)
```

Comment:
```suggestion
  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename BCs, typename Coll>
  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb)
```

Reply:
Ah, I did not know that was enough like this. It is indeed less verbose, I will change it in the next commit

Reply:
Thanks